src/server.ts

```diff
@@ -2,17 +2,23 @@ import dotenv from 'dotenv'
 dotenv.config()
 
 import express from 'express'
-import { connectDatabase } from './app/hooks/database'
+import { connectDatabase, getCollection } from './app/hooks/database'
 
 if (!process.env.MONGODB_URI) {
   throw new Error('No MongoDB URL dotenv variable')
 }
 
 const app = express()
-const port = process.env.PORT || 3001
+const port = process.env.PORT || 3000
```

Comment:
I guess your frontend is served via 3000, so your backend has to be on 3001?

Reply:
I think both must be on 3000?!